features/security/src/main/feature/feature.xml

```diff
@@ -374,6 +374,19 @@
         <bundle>mvn:ddf.security.handler/security-handler-oidc/${project.version}</bundle>
     </feature>
 
+    <feature name="security-handler-oauth" version="${project.version}"
```

Comment:
📘 This is ~ the same feature set for similar bundles; for instance the `OidcHandler`.